internal/lifecycle/readme_test.go

```diff
@@ -193,40 +194,42 @@ var toCommandsTests = []toCommandsTest{
 
 func TestToCommands(t *testing.T) {
 	for i, tc := range toCommandsTests {
-		if len(tc.codeBlock) == 0 {
-			continue
-		}
+		t.Run(fmt.Sprintf("#%d", i), func(t *testing.T) {
```

Comment:
Instead of just giving a number here, your first arg to `T.Run` should be a test case description. Add a name field to `toCommandsTest` and fill that with a brief description, replacing the comments that you have added to each case. That way when there's a failure it's clear from the output which test case failed.

Reply:
Resolved, thank you. Does the updated version have the description format you were looking for?